scala-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/scala/BaseControlClient.scala

```diff
@@ -0,0 +1,62 @@
+package com.typesafe.conductr.clientlib.scala
+
+import java.net.URL
+
+import com.typesafe.conductr.HttpPayload
+import com.typesafe.conductr.clientlib.scala.models._
+import com.typesafe.conductr.scala.AbstractConnectionContext
+
+import scala.concurrent.Future
+
+private[scala] abstract class BaseControlClient(conductrAddress: URL) {
```

Comment:
I've renamed the `BaseControlClient` to `AbstractControlClient`. The `AbstractControlClient` in the `v1` package is now renamed to `AbstractControlClientV1`.

Now:
- `AbstractControlClientV1` extends `AbstractControlClient`
- `akka.v1.ControlClient` extends `AbstractControlClientV1`
- `akka.v2.ControlClient` extends `AbstractControlClientV2`